api/v1/helpers/verbs/doFind.js

```diff
@@ -68,13 +68,23 @@ function doFindAndCountAll(reqResNext, props, opts) {
  */
 function doFindAll(reqResNext, props, opts) {
```

Comment:
That was merely an oversight--all the same filtering should apply there too (OK as separate story if it would add too much to this story)